stable/artifactory-ha/values.yaml

```diff
@@ -0,0 +1,191 @@
+# Default values for artifactory-ha.
+# This is a YAML-formatted file.
+# Beware when changing values here. You should know what you are doing!
+# Access the values with {{ .Values.key.subkey }}
+
+# Common
+initContainerImage: "alpine:3.6"
+
+# For supporting pulling from private registries
+imagePullSecrets:
+
+# Database
+## Configuration values for the postgresql dependency
+## ref: https://github.com/kubernetes/charts/blob/master/stable/postgressql/README.md
+##
+postgresql:
+  enabled: true
+  postgresDatabase: "artifactory"
+  postgresUser: "artifactory"
+  postgresPassword: "artIfakt0ri1973"
```

Comment:
By default Postgres uses auto-generated credentials. Please keep it that way and remove these defaults.